<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlideshowWindow</class>
 <widget class="QMainWindow" name="SlideshowWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="SlideshowLabel">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>600</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: black;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
   <widget class="QSlider" name="channel_slider">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>40</y>
      <width>300</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slice_slider">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>68</y>
      <width>300</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="time_slider">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>96</y>
      <width>300</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>21</y>
      <width>300</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <zorder>SlideshowLabel</zorder>
   <zorder>time_slider</zorder>
   <zorder>slice_slider</zorder>
   <zorder>label</zorder>
   <zorder>channel_slider</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuControls">
    <property name="title">
     <string>Controls</string>
    </property>
    <addaction name="actionShow_Hide"/>
   </widget>
   <addaction name="menuControls"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionShow_Hide">
   <property name="text">
    <string>Show / Hide</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
